demo/src/relying_party_frontend/src/lib/components/BuildConsentMessage.svelte

```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+	import type { IcpWallet } from '@dfinity/oisy-wallet-signer/icp-wallet';
+	import { isNullish, nonNullish } from '@dfinity/utils';
+	import { fade } from 'svelte/transition';
+	import Button from '$core/components/Button.svelte';
+	import Value from '$core/components/Value.svelte';
+	import { accountsStore } from '$lib/stores/accounts.store';
+	import { authStore } from '$core/stores/auth.store';
+	import type { Icrc1TransferRequest } from '@dfinity/ledger-icp';
+	import { E8S_PER_ICP } from '$core/constants/app.constants';
+
+	type Props = {
+		wallet: IcpWallet | undefined;
+	};
+
+	let { wallet }: Props = $props();
+
+	const INVALID_LEDGER_CANISTER_ID = import.meta.env.VITE_SATELLITE_ID;
+	console.log(INVALID_LEDGER_CANISTER_ID);
+
+	const onclick = async () => {
+		// TODO: handle errors
+		if (isNullish($authStore.identity)) {
+			return;
+		}
+
+		if (isNullish($accountsStore)) {
+			return;
+		}
+
+		const account = $accountsStore?.[0];
```

Comment:
the coalescing `?` is not needed here, right?

Reply:
Good catch. I actually removed the above `isNullish`. Did so also in the similar other component `CallCanister` (this component was copy/pasted from it).